### ISSUE TITLE
home-manager usage possible?

### ISSUE THREAD

**Issue by sigmonsays** [noamraph/nixsa#12]
Attempting to install home-manager using the standalone steps ( https://nix-community.github.io/home-manager/index.xhtml#sec-install-standalone ) does not work

```
sig@dev1:~$ nix-shell '<home-manager>' -A install
error: file 'home-manager' was not found in the Nix search path (add it using $NIX_PATH or -I)
```

Is there something i'm missing here?

**Comment by noamraph** [comment]
Thanks for the report! I reproduced this.

**Comment by noamraph** [comment]
An easy way to see the problem:
```
❯ nixsa/bin/nix eval --impure --expr builtins.nixPath
[ { path = "/home/noamyr/.nix-defexpr/channels"; prefix = ""; } ]
```
It should show `nixsa/state/defexpr/channels`, not a path in the home directory.

I still don't understand why it's so - `nix-channel` updates the expected files.

**Comment by Drakeguard** [comment]
for me it is working with:
❯ mkdir -p ~/.local/state/nix/profiles
❯ nixsa/bin/nix run home-manager/master -- init --switch

**Comment by noamraph** [comment]
@sigmonsays I released a new version of Nixsa, 0.1.3, which uses a newer version of Nix. From what I see, it now works fine. Can you verify, and re-open the issue if there's still a problem?

**Comment by sigmonsays** [comment]
FYI: I do not have permissions to reopen
